### DISCUSSION TITLE
Getting e2e tests to run reliably locally

### DISCUSSION THREAD

**Discussion by ronaldlangeveld** [facebook/lexical#4591] [General]
I have a [pending PR, with a LGTM 🎉](https://github.com/facebook/lexical/pull/4461), however it's getting bottled necked by CI tests failing, preventing it to be merged.
I'm trying to reproduce it locally, but haven't been able to test reliably (even from main branch, with exact code with CI passing).
As I understand the instructions are, run dev environment locally in one terminal and then concurrently run e2e testing in another.

Tests run but there's always some failing.

I'm not sure if I'm missing a step in between or if they are just flaky?

<img width="1793" alt="Screenshot 2023-06-02 at 15 27 08" src="https://github.com/facebook/lexical/assets/19263291/f42579ce-2556-4fef-a003-52c8028c8ca2">

Above is testing the [latest commit](https://github.com/facebook/lexical/commit/73ac48864cdd1021502a39421022ccafc2d98648) which passes on CI but cannot get locally to pass. This is of course without my code added.

So I run `npm run dev` in one terminal and then `npm run test-e2e-chromium` in another.
I tried all the e2e variations, firefox, webkit. 

I have no issues with unit tests.

System specs: Macbook M2 Pro, MacOS Ventura 13.4, Node v18.16.0.

Advice would be much appreciated!

**Comment by acywatson** [comment]
Weird - some of the tests are a bit flaky, but there shouldn't generally be any failing (might retry once) consistently locally. 

Did you look at the output of the failures to see if there are any clues? I believe screen recordings should also be created in a local directory for failed tests. They might provide some insight as well.
